src/main/modules/document-download/index.ts

```diff
@@ -36,6 +48,16 @@ export class DocumentDownloadMiddleware {
       target: config.get('services.documentManagement.url'),
     };
 
+    const dmStoreProxyForDeemedAsServicePdf = {
+      endpoints: ['/downloads/certificate-of-deemed-as-service'],
+      target: config.get('services.documentManagement.url'),
```

Comment:
i cleaned this up as i thought it was repetitive to have the target the same for all objects. ill send you what i did.